fsw/cfe-core/src/msg/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+##################################################################
+#
+# cFE message module CMake build recipe
+#
+# This CMakeLists.txt adds source files for
+# message module included in the cFE distribution.  Selected
+# files are built into a static library that in turn
+# is linked into the final executable.
+#
+# Note this is different than applications which are dynamically
+# linked to support runtime loading.  The core applications all
+# use static linkage.
+#
+##################################################################
+
+# Add the basic set of files which are always built
+set(CFE_ALL_MODULE_SRCS ${CFE_ALL_MODULE_SRCS}
+    ${CMAKE_CURRENT_SOURCE_DIR}/cfe_msg_checksum.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/cfe_msg_fc.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/cfe_msg_time.c
+    PARENT_SCOPE
+)
```

Comment:
This is very unusual to say the least -- Why wouldn't this just do an `add_library` here?  I've never seen a CMake file add sources to its parent like this.  At the very minimum, you could make an OBJECT lib and then include the objects in the lib in the parent.  This is how OSAL does it.

Reply:
Replaced with more standard implementation in b5c8be6, also messed with includes to make it easier for SB to depend on MSG (or custom MSG) once it moves to it's own module.